src/core/geometry/path/fromPointArray.test.js

```diff
@@ -0,0 +1,23 @@
+const fromPointArray = require('./fromPointArray')
+const toPointArray = require('./toPointArray')
+const test = require('ava')
+const vec3 = require('../../math/vec3')
+
+test('fromPointArray: Creating a path from no points produces an empty, non-canonical path', t => {
```

Comment:
I think a 0 length path is actually canonical but that is another debate.

Reply:
Having 0 length paths canonical is fine, if you prefer.

Reply:
Latest changes set isCanoncial = false for all functions that create new paths, including create()